t.sentinel.import/t.sentinel.import.py

```diff
@@ -457,6 +489,11 @@ def main():
             }
             if options["extent"] == "region":
                 import_kwargs["region"] = currentregion
+            if flags["c"]:
+                if options["stvds_clouds"]:
+                    import_kwargs["cloud_output"] = "vector"
+                elif options["strds_clouds"]:
+                    import_kwargs["cloud_output"] = "raster"
```

Comment:
Only raster-option needed, since `vector` is default? But also not bad to have it explicit here. In this case maybe add also the case of only c flag without output-naming-option?

Reply:
Yes, `vector` is default, now I changed it so that all `raster`/`vector` is passed explicitly in all cases